### PR TITLE
#clusters per layer and ring - TProfile2D

### PR DESCRIPTION
**Pull request by gflouris**
Change NumberOfClusterPerLayerTrendVar and NumberOfClusterPerRingVsTrendVar to TProfile2D.

### CODE REVIEW

**Comment by dmitrijus** [comment]
please test

**Comment by dmitrijus** [comment]
+1

File: DQM/SiStripMonitorCluster/src/SiStripMonitorCluster.cc [hunk 1]
```diff
@@ -785,40 +788,45 @@ void SiStripMonitorCluster::analyze(const edm::Event& iEvent, const edm::EventSe
...
-	  std::map<std::string, SubDetMEs>::iterator iSubdet  = SubDetMEsMap.find(subdet_label);
-	  std::pair<std::string,int32_t> det_layer_pair = folder_organizer.GetSubDetAndLayer(detid, tTopo);
-          if(iSubdet != SubDetMEsMap.end()) iSubdet->second.SubDetNumberOfClusterPerLayerTrend->Fill(trendVar,abs(det_layer_pair.second));
-        }
 
         if (subdetswitchtotclusprofon && (subdet_label.find("TID")!=std::string::npos || subdet_label.find("TEC")!=std::string::npos)){
-	  std::pair<std::string,int32_t> det_ring_pair = folder_organizer.GetSubDetAndLayer(detid,tTopo,true);
-          fillME(layer_single.LayerNumberOfClusterPerRingTrend, trendVar, abs(det_ring_pair.second));
+          std::pair<std::string,int32_t> det_ring_pair = folder_organizer.GetSubDetAndLayer(detid,tTopo,true);
+          ncluster_ring[abs(det_ring_pair.second)] ++ ;
```
Comment (davidlange6) [inline DQM/SiStripMonitorCluster/src/SiStripMonitorCluster.cc]: hi @gflouris - please use std::abs

Comment (gflouris) [inline DQM/SiStripMonitorCluster/src/SiStripMonitorCluster.cc]: @davidlange6 done.

**Comment by boudoul** [comment]
Hi @dmitrijus , could you please review this PR ? Thanks

**Comment by davidlange6** [comment]
please test

**Comment by davidlange6** [comment]
merge

**Comment by threus** [comment]
@gflouris could you please back port this PR to 92X? It is needed for the online DQM, otherwise your changes don't get merged into the current production branch.
Thanks

**Pull request merged**